wpeverest/everest-forms
# Feature - Repeater Fields Add-on

### All Submissions:






### Changes proposed in this Pull Request:

Previously there was no such field as **Repeater Field** which implies the repeater feature on the fields. This PR helps to implement an **Add-on** feature that helps to enhance the form features by enabling the **Repeater Field**.

### How to test the changes in this Pull Request:

1. Go to the **Plugins** menu on your Site.
2. Activate the plugin named **Everest Forms - Repeater Fields**.
3. Now open a form and go to the **Form Builder** area.
4. Drag and Drop the **Repeater Fields** field into the required place and add additional fields to acts as a repeater field inside the **Repeater** container.
5. For additional options to apply to the **Repeater Fields**, click on the **Field Options** tab and choose the required options for the repeater fields.
6. Save the form and click on the Preview button to see the changes to the form for **Repeater Fields**.
7. Now you can replicate the repeater field for **Add More** functionality in your form.
8. That's all. **Repeater Field** has been successfully applied to your form.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Repeater Fields Add-on.
